libipl/p10/ipl0.C

```diff
@@ -387,9 +388,27 @@ static void process_guard_records()
 		return;
 	}
 
-	openpower::guard::libguard_init(false);
+	openpower::guard::GuardRecords records;
+	try {
+		openpower::guard::libguard_init(false);
+		records = openpower::guard::getAll();
+	} catch (const openpower::guard::exception::InvalidGuardFile &ex) {
+		// capture the exception to use it in the PEL
+		// But continue to boot
+		ipl_log(IPL_ERROR,
+			"Caught the exception: %s and continuing to boot",
+			ex.what());
+		ipl_error_callback(IPL_ERR_INVALID_GUARD_FILE);
+		return;
+	} catch (const openpower::guard::exception::GuardException &ex) {
+		// For any other exeption, just log the trace and continue to
+		// boot
+		ipl_log(IPL_ERROR,
+			"Caught the exception %s and continuing to boot",
```

Comment:
JS: "Caught the exception: %s and continuing to boot without processing guard records"